components/core/fileminio.py

```diff
@@ -0,0 +1,40 @@
+from flask import Response, request
+from DownloadDaemon import *
+from minio import Minio
+from minio.error import ResponseError
+from DownloadManager import *
+from DBCon import *
+import sys
+import sqlalchemy.pool as pool
+
+client = Minio('localhost:9000', secure=False,
+               access_key='bassa',
+                  secret_key='bassa123')
+
+def upload_to_minio(file_name, file_path):
+	fname = file_name
+	fpath = file_path
+	try:
+		client.fput_object('bassa', fname, fpath)
```

Comment:
Can you please use function arguments here and wherever possible?

Reply:
Exactly @demon-36 :) As these variables are pass by value so no sense of copying again